assets/js/pages/HostsList/HostsList.jsx

```diff
@@ -119,22 +119,18 @@ function HostsList() {
         filter: (filter, key) => (element) =>
           element[key].some((sid) => filter.includes(sid)),
         render: (sids, { sap_systems }) => {
-          const sidsArray = uniqBy(sap_systems, getInstanceID).map(
-            (instance, index) => {
-              const instanceID = getInstanceID(instance);
-              return [
-                index > 0 && <br />,
-                <SapSystemLink
-                  key={`${instanceID}-${instance?.id}`}
-                  systemType={instance?.type}
-                  sapSystemId={instanceID}
-                >
-                  {instance?.sid}
-                </SapSystemLink>,
-              ];
-            }
-          );
-          return sidsArray;
+          const sidsArray = uniqBy(sap_systems, getInstanceID);
+          return sidsArray.map((instance, index) => (
+            <span key={`${sids[index]}-${instance?.id}`}>
+              <SapSystemLink
+                key={`${getInstanceID(instance)}-${instance?.id}`}
+                systemType={instance?.type}
+                sapSystemId={getInstanceID(instance)}
+              >
+                {instance?.sid}
+              </SapSystemLink>{' '}
+            </span>
+          ));
```

Comment:
I'm not sure a PR introducing some tests should actually modify features' code too :smile: 


This refactor is just made to remove the line break right?

Reply:
Did the small refactor to remove the line break, it produced some issues with existing tests when additional sid was added. 
I guess i can revert it, but i may to modify existing tests more. 
What do you think ?

Reply:
This change have some ui changes? Like in the appaerence of things

Reply:
No it looks the same and that is the fun thing about it :+1: